doc/reference.rst

```diff
@@ -42,3 +53,215 @@ It then runs the resulting buildout.
 
 See :ref:`Bootstrapping <init-generates-buildout.cfg>` for examples.
 
+
+.. _buildout-configuration-options-reference:
+
+Buildout configuration options
+===============================
+
+The standard buildout options are shown below.  Values of options with
+defaults shown can be used in :ref:`value substitutions
+<value-substitutions>`.
+
+abi-tag-eggs
+  A flag (true/false) indicating whether the eggs directory should be
+  divided into subdirectories by `ABI tag
+  <https://www.python.org/dev/peps/pep-0425/#abi-tag>`_.  This may be
+  useful if you use multiple Python builds with different build
+  options or different Python implementations.  It's especially
+  useful if you switch back and forth between PyPy and C Python.
+
+allow-hosts, default: '*'
+  Specify which hosts (as globs) you're willing to download
+  distributions from when following :ref:`dependency links
+  <use-dependency-links>`.
+
+allow-picked-versions, default: 'true'
+  Indicate whether it should be possible to install requirements whose
+  `versions aren't pinned <pinned-versions>`.
+
+bin-directory, default: bin
+  The directory where generated scripts should be installed. If this
+  is a relative path, it's evaluated relative to the buildout
+  directory.
+
+.. _develop-option:
+
+develop
+  One or more paths to `distutils setup scripts
+  <https://docs.python.org/3.6/distutils/setupscript.html>`_ or (more
+  commonly) directories containing setup scripts named ``setup.py``.
```

Comment:
Might be worth mentioning how to separate multiple paths (whitespace, or just newlines?).